src/libaktualizr/uptane/managedsecondary.cc

```diff
@@ -1,11 +1,14 @@
-#include "uptane/managedsecondary.h"
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
 
 #include <boost/algorithm/hex.hpp>
 #include <boost/filesystem.hpp>
 
+#include "uptane/managedsecondary.h"
+
```

Comment:
Normally I think we prefer a `.cc`'s `.h` file to be the first header file listed. I think that's what the Google Style Guide recommends, too.